pom.xml

```diff
@@ -2957,7 +2957,7 @@
               <JAVA_HOME>${test.java.home}</JAVA_HOME>
               <SPARK_BEELINE_OPTS>-DmyKey=yourValue</SPARK_BEELINE_OPTS>
             </environmentVariables>
-            <systemProperties>
+            <systemPropertyVariables>
```

Comment:
Are you using maven 3.9.0? There should be no this warning log in 3.8.7.

Reply:
Yes,
<img width="461" alt="image" src="https://user-images.githubusercontent.com/15246973/222961036-5568645a-c681-4889-b7cf-08a2f1ecb0fc.png">

<img width="871" alt="image" src="https://user-images.githubusercontent.com/15246973/222961017-5e1e4cf8-19a3-44f8-8484-6791b66d34d3.png">

Reply:
I think we don't  need to make this change for the time being. Apache Spark has not promised to use maven 3.9.0 for build and test

Reply:
And I think we should do this change after finish SPARK-42380

Reply:
Does it hurt to make this change now? still works in Maven 3.8 and just prepares for 3.9?

Reply:
In fact, I think it can work well in maven 3.8.
Because`systemProperties` has been marked as `Deprecated` long ago.